easybuild/easyblocks/o/openfoam.py

```diff
@@ -504,27 +510,49 @@ def sanity_check_step(self):
                 else:
                     geom_target_dir = 'triSurface'
 
+            if self.looseversion < LooseVersion('11'):
+                cmds = [
+                        "cp -a %s %s" % (motorbike_path, test_dir),
+                        # Make sure the tmpdir for tests ir writeable if read-only-installdir is used
+                        "chmod +w  %s" % test_dir,
```

Comment:
need to make sure whole test dir is writable:
```suggestion
                        "chmod -R +w %s" % test_dir,